src/Domain/Item/SyncResponse/SyncResponseFactory20161215.ts

```diff
@@ -27,12 +35,47 @@ export class SyncResponseFactory20161215 implements SyncResponseFactoryInterface
     }))
 
     return {
-      retrieved_items: syncItemsResponse.retrievedItems.map(item => <ItemProjection> this.itemProjector.projectFull(item)),
+      retrieved_items: pickOutConflictsResult.retrievedItems.map(item => <ItemProjection> this.itemProjector.projectFull(item)),
       saved_items: syncItemsResponse.savedItems.map(item => <ItemProjection> this.itemProjector.projectFull(item)),
       unsaved,
       sync_token: syncItemsResponse.syncToken,
       cursor_token: syncItemsResponse.cursorToken,
       integrity_hash: syncItemsResponse.integrityHash,
     }
   }
+
+  private pickOutConflicts(
+    savedItems: Array<Item>,
+    retrievedItems: Array<Item>,
+    unsavedItems: Array<ItemConflict>
+  ): {
+    unsavedItems: Array<ItemConflict>,
+    retrievedItems: Array<Item>,
+  } {
+    const savedIds: Array<string> = savedItems.map((savedItem: Item) => savedItem.uuid)
+    const retrievedIds: Array<string> = retrievedItems.map((retrievedItem: Item) => retrievedItem.uuid)
+
+    const conflictingIds = savedIds.filter(savedId => retrievedIds.includes(savedId))
+
+    for (const conflictingId of conflictingIds) {
+      const savedItem = savedItems.filter(item => item.uuid === conflictingId)[0]
+      const conflictedItem = retrievedItems.filter(item => item.uuid === conflictingId)[0]
+
+      const difference = savedItem.updatedAtTimestamp - conflictedItem.updatedAtTimestamp
```

Comment:
Note that this result would be in microseconds, but LEGACY_MIN_CONFLICT_INTERVAL is in seconds.